core/src/main/scala/chandu0101/scalajs/react/components/ReactTable.scala

```diff
@@ -10,18 +10,22 @@ import scalacss.ScalaCssReact._
 
 object ReactTable {
 
-  type Model = Map[String, Any]
+  type ColumnKey = String
+  type Model = Map[ColumnKey, Any]
```

Comment:
I know! I tried, but you can't parametrize a companion object, you can parametrize everything, but that makes the file nearly illegible. I tried making it into a class, but didn't think of a trait, that may work.